### PR TITLE
Improve workflow content

### PR DESCRIPTION
**Pull request by miroslavpojer**
- Refactored release_draft.yml file content. Improved tag control logic.

### CODE REVIEW

**Comment by miroslavpojer** [comment]
Release notes:
- Improve logic controlling tag creation.

File: .github/workflows/release_draft.yml [hunk 1]
```diff
@@ -55,11 +55,14 @@ jobs:
...
 
-            const latestTag = refs.sort((a, b) => new Date(b.object.date) - new Date(a.object.date))[0].ref.replace('refs/tags/', '');
-            console.log(`Latest tag: ${latestTag}`);
+            if (refs.length === 0) {
+              // No existing tags, so any new tag is valid
+              console.log('No existing tags found. Any new tag is considered valid.');
+              return;
+            }
 
+            const latestTag = refs.sort((a, b) => new Date(b.object.date) - new Date(a.object.date))[0].ref.replace('refs/tags/', '');
```
Comment (benedeki) [inline .github/workflows/release_draft.yml:65]: This is wrong. Don't sort by date but by version expression. (if you are changing it)

File: .github/workflows/release_draft.yml [hunk 2]
```diff
@@ -75,35 +78,35 @@ jobs:
...
         with:
           tag-name: ${{ github.event.inputs.tag-name }}
-          chapters: |
-            [
-              {"title": "Breaking Changes 💥", "label": "breaking-change"},
-              {"title": "New Features 🎉", "label": "feature"},
-              {"title": "New Features 🎉", "label": "enhancement"},
-              {"title": "Bugfixes 🛠", "label": "bug"}
-            ]
+          chapters: '[
```
Comment (benedeki) [inline .github/workflows/release_draft.yml:100]: Why is the array a string?

Comment (miroslavpojer) [reply .github/workflows/release_draft.yml:100]: Will be fixed in Issue #https://github.com/AbsaOSS/generate-release-notes/issues/85.

**Pull request merged**